protocols/gossipsub/CHANGELOG.md

```diff
@@ -6,7 +6,10 @@
 
 - Update to `libp2p-swarm` `v0.42.0`.
 
+- Initialize `ProtocolConfig` via `GossipsubConfig`. See [PR 3381].
+
 [PR 3207]: https://github.com/libp2p/rust-libp2p/pull/3207/
+[PR 3381]: https://github.com/libp2p/rust-libp2p/pull/3381/
```

Comment:
Why does this need a changelog entry? This is not user facing, right?

Reply:
It is (unfortunately) part of the public API.

Reply:
Happy to remove it again if you think that doesn't warrant an entry!

Reply:
Oh, I am surprised. No strong opinion. Let's keep as is :+1:

Reply:
Yep, it's something I'd like to fix. See linked discussion in PR description.